.github/workflows/tests.yml

```diff
@@ -29,26 +29,6 @@ jobs:
     env:
       # override make command to install directly in active python
       CONDA_COMMAND: ""
-    services:
-      # Label used to access the service container
-      postgres:
-        image: postgres  # DockerHub
-        env:
-          POSTGRES_USER: postgres
-          POSTGRES_PASSWORD: qwerty
-        ports:
-          - "5432:5432"
-        # Set health checks to wait until postgres has started
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-      # Label used to access the service container
-      mongodb:
-        image: mongo:3.4.23  # DockerHub
-        ports:
-          - "27017:27017"
```

Comment:
Since the the `make docker-up-test` command would start `postgres` and `mongodb` containers, I figured those were not needed anymore, so I removed them.
At first I had tried to just add Magpie to this list, but the Magpie image would need some files from the repo, and the repo was only checked out after starting those services, hence why I added a `start containers` step later instead.